modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -59,6 +60,17 @@ When you create this API in API Manager:
 NOTE: Using this feature requires an API Manager license.
 |
 
+|HTTPS
+a|Applies to CloudHub deployments.
+
+Indicates whether Partner Manager deploys this endpoint as an HTTPS or HTTP listener:
+
+* If enabled (default), Partner Manager deploys this endpoint as an HTTPS listener.
```

Comment:
repeated bullet list - candidate for a partial?

Reply:
Maybe later